akka-http-core/src/main/scala/akka/http/impl/settings/ServerSentEventSettingsImpl.scala

```diff
@@ -0,0 +1,24 @@
+package akka.http.impl.settings
```

Comment:
please add the header: `Copyright (C) 2017-2018 Lightbend Inc. <https://www.lightbend.com>`
otherwise build will fail